docs/references/_attachments/ic.did

```diff
@@ -479,6 +479,10 @@ type delete_canister_snapshot_args = record {
 
 type fetch_canister_logs_args = record {
     canister_id : canister_id;
+    filter : opt variant {
```

Comment:
This PR should be merged when the roll-out is complete and changelog should be updated (shortly) before merging (to avoid merge conflicts).